commons/src/main/java/info/metadude/android/eventfahrplan/commons/DateFormatter.kt

```diff
@@ -0,0 +1,51 @@
+package info.metadude.android.eventfahrplan.commons
+
+import org.threeten.bp.OffsetDateTime
+import org.threeten.bp.format.DateTimeFormatter
+import java.text.SimpleDateFormat
+import java.util.*
+
```

Comment:
# import java.util.*
There are a few places in the project where `import java.util.*` is still used. Please check if they can be removed/replaced with `org.threeten....`

Reply:
Can you please comment on whether we should or not change from `java.util.Date` to a `org.threeten` type in `DateFormatter` and if there are APIs in `org.threeten` to substitute `java.util.Locale` and `java.util.TimeZone` which are used in `DateFormatterTest`?

Reply:
Locale and TimeZone have no counterpart as far as I know. 
org.threeten.bp.format.DateTimeFormatter itself requires java.util.Date.

Reply:
Thanks. Let's see if we find a substitute in the future.